src/commcare_cloud/fab/fabfile.py

```diff
@@ -402,6 +402,7 @@ def deploy_checkpoint(command_index, command_name, fn, *args, **kwargs):
 
 def _deploy_without_asking(skip_record):
     try:
+        raise PreindexNotFinished()
```

Comment:
I think you put this here just to test it and then left it in?

Reply:
Ugh yes